packages/geo/src/projection.ts

```diff
@@ -14,10 +14,30 @@ export enum EPSG {
     Nztm = 2193,
 }
 
+const EPSGTextMap: Record<string, EPSG> = {
+    google: EPSG.Google,
+    'epsg:3857': EPSG.Google,
+    '3857': EPSG.Google,
+    'global-mercator': EPSG.Google,
```

Comment:
could replace out anything that is not a string or number make the map simpler

Reply:
I don't understand what you mean?